CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/utilities/AnalyticUtilities.java

```diff
@@ -132,4 +134,15 @@ public static Collection<GraphVisualisationTranslator> lookupGraphVisualisationT
     public static GraphVisualisationTranslator<?, ?> lookupGraphVisualisationTranslator(final String graphVisualisationTranslatorName) {
         return GRAPH_VISUALISATION_TRANSLATORS.get(graphVisualisationTranslatorName);
     }
+    
+    /**
+     * Helper method to get the path for the documentation pages 
+     * @return path
+     */
+    public static String getHelpPath() {
+        final String codebaseName = "constellation";
+        final String sep = File.separator;
```

Comment:
This could be moved out of the method and defined as a class member variable:
`private static final String SEP = File.separator;`